app/assets/javascripts/angular/controllers/pages/create_recipe_controller.js.erb

```diff
@@ -48,36 +75,84 @@ app.controller('createRecipeCtrl', ['$scope', '$http', '$routeParams', function(
     }
 
     $(function() {
-      $( "#slider" ).slider({
-        // range: true,
+      $( "#prep-time-slider" ).slider({
         min: 0,
         max: 1440,
-        values: [ 0, 1440 ],
+        values: [0],
         slide: function( event, ui ) {
+          $( "#prep-time-amount" ).val( timeConversion(ui.values));
+          $scope.create_parameters.timeframe.prepTime = ui.values;
+          console.log($scope.create_parameters);
+        }
+      });
+      // initial slider values
+      $( "#prep-time-amount" ).val( timeConversion(0));
+    });
 
-          // $( "#amount" ).val( timeConversion(ui.values[0]) + " to " + timeConversion(ui.values[ 1 ]) );
+    $(function() {
+      $( "#cook-time-slider" ).slider({
+        min: 0,
+        max: 1440,
+        values: [0],
+        slide: function( event, ui ) {
+          $( "#cook-time-amount" ).val( timeConversion(ui.values));
+          $scope.create_parameters.timeframe.cookTime = ui.values;
+          console.log($scope.create_parameters);
+        }
+      });
+      // initial slider values
+      $( "#cook-time-amount" ).val( timeConversion(0));
+    });
 
-          $scope.search_parameters.timeframe = ui.values;
-          console.log($scope.search_parameters);
+    $(function() {
+      $( "#rest-time-slider" ).slider({
+        min: 0,
+        max: 1440,
+        values: [0],
+        slide: function( event, ui ) {
+          $( "#rest-time-amount" ).val( timeConversion(ui.values));
+          $scope.create_parameters.timeframe.restTime = ui.values;
+          console.log($scope.create_parameters);
         }
       });
       // initial slider values
-      // $( "#amount" ).val( $( "#slider-range" ).slider( "values", 0 ) + " - " + $( "#slider-range" ).slider( "values", 1 ) +" mins" );
-      // $( "#amount" ).val( timeConversion(0) + " to " + timeConversion(1440) );
+      $( "#rest-time-amount" ).val( timeConversion(0));
     });
+    
   }
+    
+  { // add instruction functions
+    
+    $scope.create_parameters.all_instructions =[];
```

Comment:
`$scope.create_parameters.all_instructions = []`

Please fix these things across the project